lib/rules/callback-return.js

```diff
@@ -39,7 +39,15 @@ module.exports = function(context) {
      * @returns {Boolean} Whether or not this function matches our callback name.
      */
     function isCallback(node) {
-        return node.callee.type === "Identifier" && callbacks.indexOf(node.callee.name) > -1;
+        if (node.callee.type === "Identifier" && callbacks.indexOf(node.callee.name) > -1) {
+            return true;
+        }
+
+        if (node.callee.type === "MemberExpression" && callbacks.indexOf(node.callee.object.name + "." + node.callee.property.name) > -1) {
```

Comment:
This assumes that both `object` and `property` are always identifiers, but they can be any number of types:

```js
foo.bar.baz;   // object is another MemberExpression
foo().baz;    // object is a CallExpression
```

So here we can either verify that both `object` and `property` are `Identifier` (and therefore, don't allow `foo.bar.baz` as a valid configuration), or expand this to take into account multiple nesting levels.